integration_tests/e2e/goal/createAndEditGoal.cy.ts

```diff
@@ -83,12 +83,39 @@ context('Create and edit a goal', () => {
     const reviewPage = Page.verifyOnPage(ReviewPage)
     reviewPage.clickChangeGoalTargetDateLink()
 
+    const today = new Date()
```

Comment:
Cypress test was failing because it was setting the target completion date to 26 Feb 2024 which fails the validation (target completion dates have to be in the future, and today is 26 Feb 2024 and is not in the future! 🤣 )

Changed the test to work out and use "today + 1 year", so this test should not fail in the future 🤞